projects/storefrontapp-e2e-cypress/cypress.ci.b2b.json

```diff
@@ -0,0 +1,19 @@
+{
+  "defaultCommandTimeout": 15000,
+  "baseUrl": "http://localhost:4200",
+  "requestTimeout": 15000,
+  "projectId": "k3nmep",
+  "video": true,
+  "retries": {
+    "runMode": 2
+  },
+  "env": {
+    "CLIENT_ID": "mobile_android",
+    "CLIENT_SECRET": "secret",
+    "API_URL": "https://spartacus-dev0.eastus.cloudapp.azure.com:9002",
+    "OCC_PREFIX": "/occ/v2",
+    "OCC_PREFIX_USER_ENDPOINT": "orgUsers",
+    "OCC_PREFIX_ORDER_ENDPOINT": "orders",
+    "BASE_SITE": "powertools-spa"
+  }
```

Comment:
I think we might have lost `B2B=true` in the process.

Reply:
b2b=true is not used that is why I did not include it. I remember we removed it before, but was not cleaned afterwards